packages/zoltan2/core/src/input/Zoltan2_VectorAdapter.hpp

```diff
@@ -175,6 +175,15 @@ template <typename User>
     Kokkos::deep_copy(elements, host_elements);
   }
 
+  virtual void getEntriesHostView(typename AdapterWithCoords<User>::CoordsHostView & elements) const {
+      Z2_THROW_NOT_IMPLEMENTED
+
+  }
+
+  virtual void getEntriesDeviceView(typename AdapterWithCoords<User>::CoordsDeviceView& elements) const {
+      Z2_THROW_NOT_IMPLEMENTED
+  }
+
```

Comment:
Since they're first declared here, could you provide a basic doxygen?

Reply:
done